daps-xslt/profiling/check.profiling.xsl

```diff
@@ -1,169 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
-<!-- Checks, if an element needs to be profiled
-     returns 0 (needs to be profiled) or 1 (no profiling neccessary)
+<!-- Checks if an element needs to be profiled
+     returns 0 (needs to be profiled) or 1 (no profiling necessary)
 
-     Taken from the original DocBook XSL Stylesheets, 
+     Taken from the original DocBook XSL Stylesheets,
      see profiling/profile-mode.xsl
 -->
-   
+
 <xsl:include href="pi-attribute.xsl"/>
 
 <xsl:param name="provo.root"/>
 
 <xsl:template name="check.profiling">
   <xsl:param name="context" select="."/>
+  <xsl:param name="attributes" select="$profiling.attributes.enabled"/>
+  <xsl:param name="values" select="$profiling.values.merged"/>
```

Comment:
As I said, my changes were focused on `list-src-files`... All else, including profiling, I will have to fix later on.